index.html

```diff
@@ -1044,48 +1060,101 @@
             <li>Return <var>P</var>, but continue running these steps in
```

Comment:
Spec bug: These steps should be aborted in the case of a SecurityError.   (I can fix this in a later PR.)